.github/workflows/ci.yml

```diff
@@ -164,8 +164,8 @@ jobs:
       - name: Install linux dependencies.
         if: ${{ matrix.spec.name == 'Linux' }}
         run: sudo apt-get install xvfb
-      - name: Build
-        run: npm run build
+      - name: Build tests
```

Comment:
What's the reason to build tests only instead of the full build?

Reply:
A production build is now separate from a test build. Test builds do not generate production artifacts.